### ISSUE TITLE
Size of samples on Teensy LC

### ISSUE THREAD

**Issue by dyfer** [sensorium/Mozzi#76]
Hello,

First off, thanks to all developers for a wonderful library!

I've been trying to use it with Teensy LC. After scaling the audio data to 12-bit range, I can get a sound from a simple sample playback and an oscillator. 

However, if I try to load a sample larger than 8kB, Teensy seems to freeze once I initialize `Sample` with that data. My understanding is that samples are read directly from flash, is that not the case? Teensy LC has 8kB RAM and 62kB flash, according to the specs.

**Comment by tfry-git** [comment]
Yes, samples should be read directly from flash (on MCUs other than ESP8266, at least), I have no good idea what might be the problem.

Some poking in the dark, perhaps one of the following questions will result in a useful pointer:
Are you sure that 8kB is the limit? Have you tried 10kB for instance? What happens if you use the burroughs-data but limit the number of cells in use (i.e. ```Sample <8192, AUDIO_RATE> aSample(BURROUGHS1_18649_DATA);``` or similar)?
